### ISSUE TITLE
rethink-migrate is stale

### ISSUE THREAD

**Issue by vinicius0026** [JohanObrink/rethink-migrate#36]
Unfortunately, this tool has become stale. There are some very old issues and pull requests open, and no response from the repo owner.

To solve some issues with this lib, I started a very similar project [`rethinkdb-migrate`](https://github.com/vinicius0026/rethinkdb-migrate). By now, the new lib has feature parity with this one, so anyone using `rethink-migrate` can try `rethinkdb-migrate` without much hassle.

@JohanObrink if you want to join the two libs, I am open to this idea.

@ilja903 @snmgian @ibolmo @jordanh @BBB @busterc I'm tagging you because you have open PR's in `rethink-migrate`. Sorry for any inconvenience.

**Comment by JohanObrink** [comment]
Yes, I am definitely open to that. Since I am suffering from a lack of time (which is why the project has become stale) I don't feel that I can drive the merge. But I can help you as far as I can. If you want to ( @vinicius0026 @ilja903 @ibolmo @jordanh @BBB @busterc ) I can ad you all as contributors to the project so that we can control it together. I am also open to migrating the repo to a common organization with equal control.
